### PR TITLE
psammead-locale bug fix for IE11

### LINKED ISSUE #2005: IE11 does not support multi-line split psammead-locale
**Describe the bug**
IE11 does not support just the multi-line split locale from psammead-locales

**To Reproduce**
This was highlighted in https://github.com/bbc/simorgh/pull/3468/ where stroybook failed to load in IE11

1. View simorgh's  storybook on IE11 or  Run chromaticqa locally 

**Expected behaviour**
Storybook should be able to evaluate and render properly in IE11 while using `@bbc/psammead-locales/moment/ar`

**Screenshots**
Screenshot 2019-09-05 at 11 45 56 AM

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
Blocks https://github.com/bbc/simorgh/pull/3468

### PR DESCRIPTION
**Pull request by RayNjeri**
Signed-off-by: Rachael Njeri <rachael.njeri@andela.com>

Resolves #2005

**Overall change:** _Disables eslint rules on multi-line split for IE11 support_

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-locales/moment/ps.js [hunk 1]
```diff
@@ -18,9 +19,7 @@ const pashtoJalaliMonths = [
   'کب',
 ];
```
Comment (EinsteinNjoroge) [inline packages/utilities/psammead-locales/moment/ps.js:20]:
```suggestion
// eslint-disable-next-line prettier/prettier
```

**Review by EinsteinNjoroge — APPROVED** [review]
Otherwise looks good to me

File: packages/utilities/psammead-locales/moment/ps.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```
Comment (jamesdonoh) [inline packages/utilities/psammead-locales/moment/ps.js]: Did you mean to disable for whole file?

Comment (RayNjeri) [reply packages/utilities/psammead-locales/moment/ps.js]: oops, fixing it

**Review by jamesdonoh — APPROVED** [review]
Thanks for the speedy fix.

**Review by dr3 — APPROVED** [review]

**Comment by dr3** [comment]
Need to admin merge, so we can pull this fix into https://github.com/bbc/psammead/blob/latest/packages/containers/psammead-timestamp-container/src/index.stories.jsx#L14

**Pull request merged**